applications/ContactStructuralMechanicsApplication/custom_conditions/mesh_tying_mortar_condition.cpp

```diff
@@ -1287,18 +1270,18 @@ boost::numeric::ublas::bounded_matrix<double, 12, 12> MeshTyingMortarCondition<2
 
 template< >
```

Comment:
you have twice template<>

Reply:
Because there are two template definition, the method and the class